### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by jdorfman**

### CODE REVIEW

**Comment by jdorfman** [comment]
@tzuryby I need help with the failed test.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+# 📑 Submitting a Pull Request (PR)
+
+* Fork the repo.
+
+* Create your PR. 
+
+* Tests will automatically run for you.
+
+* We will **NOT** merge any PR that is not passing tests.
```
Comment (flaper87) [inline CONTRIBUTING.md:19]: Wondering if we could skip this line. I think it's safe to assume this behavior. Mentioning it (with emphasis in NOT) may come across the wrong way.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
+👋 Hey there fellow open source contributor! Please read the following guidelines below carefully to maximize the chances of your PR being merged. ⤵️
+
+# 💬 Communication
+
+* ⚠️ **Before starting work on a major feature**, please reach out to us via [GitHub Discussions](https://github.com/curiefense/curiefense/discussions/categories/q-a), [Twitter DM](https://twitter.com/messages/compose?recipient_id=1321123258993938435),
```
Comment (flaper87) [inline CONTRIBUTING.md:5]: Wondering how well the `twitter DM` option has worked out so far. It's a bit uncommon to use twitter as a platform for contributors to reach out to the project. Perhaps focusing on GH Issues only would be better. It helps to keep the history in one place.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jdorfman) [reply CONTRIBUTING.md:19]: Good call, on it

File: CONTRIBUTING.md (hunk 2 above)
Comment (jdorfman) [reply CONTRIBUTING.md:5]: Another good call. On it.

**Comment by Aviv-Galmidi** [comment]
I've pulled the latest changes from master into this branch. The tests pass now

**Pull request merged**